package.json

```diff
@@ -48,7 +52,6 @@
         "react-native-onboarding-swiper": "https://github.com/neuland-ingolstadt/react-native-onboarding-swiper.git",
         "react-native-reanimated": "3.3.0",
         "react-native-root-toast": "^3.4.1",
-        "react-native-safe-area-context": "4.6.3",
```

Comment:
expo-router requires this package.

After double checking and removing `node_modules` and reinstalling everything, I get the following error:

```shell
Android Bundling failed 2211ms
Unable to resolve "react-native-safe-area-context" from "node_modules\expo-router\build\ExpoRoot.js"
```